app/build.gradle

```diff
@@ -25,11 +26,16 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
 }
 
 dependencies {
     implementation libraries.androidxAppCompat
     implementation libraries.androidxConstraintLayout
+    implementation libraries.androidxNavigationFragmentKtx
+    implementation libraries.androidxNavigationUiKtx
```

Comment:
It wasn't clear to me from the guide which pieces are in which artifact. I don't care at this point, I suppose.